nemo_text_processing/text_normalization/ja/taggers/date.py

```diff
@@ -65,8 +65,8 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
 
         graph_cardinal = cardinal.just_cardinals
 
-        month = pynini.string_file(get_abs_path("data/date/month.tsv"))
-        day = pynini.string_file(get_abs_path("data/date/day.tsv"))
+        month = pynutil.delete("0") + pynini.string_file(get_abs_path("data/date/month.tsv"))
```

Comment:
Will this also work if the digits aren't preceded by `0`s? Would it be useful to make `pynutil.delete("0").ques`?